bedrock/firefox/templates/firefox/welcome/page4.html

```diff
@@ -0,0 +1,132 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import send_to_device with context %}
+{% from "macros-protocol.html" import hero with context %}
+
+{% add_lang_files "firefox/sendto" "firefox/welcome/page4" %}
+
+{% extends "firefox/welcome/base.html" %}
+
+{# L10n: HTML page title #}
+{% block page_title %}{{ _('Download the Firefox Browser on your Mobile for iOS and Android') }}{% endblock %}
+
+{% block body_class %}{{ super() }} welcome-page4{% endblock %}
+
+{% set show_send_to_device = LANG in settings.SEND_TO_DEVICE_LOCALES %}
+
+{% set _entrypoint = 'mozilla.org-firefox-welcome-4' %}
+{% set _utm_campaign = 'welcome-4-mobile' %}
+
+{# L10n: the strong tag here is to underline Firefox. #}
+{% block content_intro %}
+{% call hero(
+    title=_('Wallet. Keys. Phone. <strong>Firefox.</strong>'),
+    desc=_('Take privacy with you on every device — and leave the data trackers behind.'),
+    class='mzp-t-firefox mzp-t-dark',
+    include_cta=True,
+    heading_level=1
+  ) %}
+
+  <p class="primary-cta">
+    <button class="js-modal-link mzp-c-button mzp-t-product" data-cta-position="primary" rel="external noopener">
+      {{ _('Get the Firefox App') }}
+    </button>
+  </p>
+
+  {% endcall %}
+
+  <div id="modal" class="mzp-u-modal-content mzp-l-content">
```

Comment:
Moving this inside the hero might improve the experience a little for users running noscript:

![image](https://user-images.githubusercontent.com/400117/69135868-be838180-0ab1-11ea-8957-234ab58152c0.png)

Reply:
Thanks for the feedback, Alex! I consulted with the designer on this page (Cristina) for what it should look like for noscript users. 👍 She told me to opt for outside the hero.

Reply:
Ok, it would be nice to center the form then?

![image](https://user-images.githubusercontent.com/400117/69163483-2fdd2780-0ae6-11ea-8846-c7c8332d1478.png)

Reply:
Whoops! Agreed, will add a fix. 🙂 thank you!